extensions/wikia/AdEngine/js/template/porvata.js

```diff
@@ -126,6 +131,22 @@ define('ext.wikia.adEngine.template.porvata', [
 			}
 		});
 
+		if (params.slotNameForFallbackBid) {
```

Comment:
Please consider using boolean flag for enabling fallbackBid and using `params.slotName`